src/platform/intel/cavs/lib/memory.c

```diff
@@ -23,6 +23,8 @@ extern uintptr_t _buffer_heap, _sof_core_s_start;
 #if CONFIG_CORE_COUNT > 1
 extern uintptr_t _runtime_shared_heap, _system_shared_heap;
 #endif
+extern uint32_t _runtime_heap_size;
```

Comment:
`size_t`?

Reply:
let me change to calculate this size locally.